examples/nirfsg/attenuation-tables-generation.py

```diff
@@ -0,0 +1,159 @@
+
+import math
+import sys
+
+import grpc
+import nidevice_pb2 as nidevice_grpc
+import nirfsg_pb2 as nirfsg_types
+import nirfsg_pb2_grpc as grpc_nirfsg
+import numpy as np
+
+SERVER_ADDRESS = "localhost"
+SERVER_PORT = "31763"
+SESSION_NAME = "NI-RFSG-Session"
+
+# Resource name, channel name and options for a VST client.
+RESOURCE = "5840_1"
+OPTIONS = ""
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    SERVER_ADDRESS = sys.argv[1]
+if len(sys.argv) >= 3:
+    SERVER_PORT = sys.argv[2]
+if len(sys.argv) >= 4:
+    RESOURCE = sys.argv[3]
+    OPTIONS = ""
+# Create a gRPC channel + client.
+channel = grpc.insecure_channel(f"{SERVER_ADDRESS}:{SERVER_PORT}")
+client = grpc_nirfsg.NiRFSGStub(channel)
+vi = None
+
+
+def raise_if_error(response):
+    """Raise an exception if an error was returned."""
+    if response.status != 0:
+        response = client.ErrorMessage(nirfsg_types.ErrorMessageRequest(error_code=response.status))
+        raise Exception(f"Error: {response.error_string}")
+
+
+def raise_if_initialization_error(response):
+    """Raise an exception if an error was returned from Initialize."""
+    if response.status < 0:
+        raise RuntimeError(f"Error: {response.error_message or response.status}")
+    if response.status > 0:
+        sys.stderr.write(f"Warning: {response.error_message or response.status}\n")
+    return response
+
+
+try:
+    response = client.InitWithOptions(
+        nirfsg_types.InitWithOptionsRequest(
+            session_name=SESSION_NAME, resource_name=RESOURCE, option_string=OPTIONS
+        )
+    )
+    raise_if_initialization_error(response)
+    vi = response.vi
+
+    client.ConfigureRefClock(
+        nirfsg_types.ConfigureRefClockRequest(
+            vi=vi,
+            ref_clock_source_mapped=nirfsg_types.REF_CLOCK_SOURCE_ONBOARD_CLOCK,
+            ref_clock_rate=10000000,
+        )
+    )
+
+    client.SetAttributeViString(
+        nirfsg_types.SetAttributeViStringRequest(
+            vi=vi,
+            channel_name="",
+            attribute_id=nirfsg_types.NIRFSG_ATTRIBUTE_REF_CLOCK_SOURCE,
+            value_mapped=nirfsg_types.NIRFSG_STRING_REF_CLOCK_SOURCE_ONBOARD_CLOCK,
+        )
+    )
+
+    raise_if_error(
+        client.ConfigureRF(nirfsg_types.ConfigureRFRequest(vi=vi, frequency=1e9, power_level=-10))
+    )
+
+    # Having a table of attenuations per frequency, we demo how to convert them to S parameters
+    # and download them to RFSG to be automatically set when we change the frequency
+    frequency_table_hz = [1e9, 6e9]
+    attenuation_table_db = [3, 3]
+
+    # Convert to S parameters: Linear attenuation
+    s = [math.pow(10, -x / 20) for x in attenuation_table_db]
+    # We ned a table that it is frequencies x 2 x 2. Easier to fill it as a 3D matrix
+    sparam = np.zeros(shape=(len(frequency_table_hz), 2, 2))
+    for index in range(len(frequency_table_hz)):
+        sparam[index, 0, 1] = s[index] #S12
+        sparam[index, 1, 0] = s[index] #S21
```

Comment:
```suggestion
        sparam[index, 0, 1] = s[index]  # S12
        sparam[index, 1, 0] = s[index]  # S21
```
The `black` formatter is complaining about missing a space between the `#` and the comment